libsplinter/src/network/connection_manager/messages.rs

```diff
@@ -21,7 +21,7 @@ pub enum CmMessage {
     Subscribe(String, Sender<Vec<CmNotification>>),
     UnSubscribe(String),
     Request(CmRequest),
-    HeartbeatNotifications(Vec<CmNotification>),
+    SendHeartbeats,
```

Comment:
Consider calling this "HeartbeatPulse". Which, I guess, might make the current HeartbeatMonitor a HeartbeatPacemaker or HeartbeatPulseGenerator.

Conceptually, this would make HeartbeatMonitor more passive -- it's not requesting heartbeats as much as providing the tempo/pulse at which they should be considered.

Reply:
I think Pacemaker actually makes a lot of sense, but I think HeartbeatPacemaker is tad redundant. Instead of "HeartbeatPacemaker" what if we just called the struct "PaceMaker"?